client/src/client.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var App = React.createClass({
+  render: function () {
+    return (
+     <p>Hello World</p>
+    )
+  }
+});
```

Comment:
This is very old way for creating React components. For creating components for future tasks i suggest you to follow this link: https://facebook.github.io/react/tutorial/tutorial.html and make it.